cmd/admin-handlers.go

```diff
@@ -985,7 +985,7 @@ func (adminAPI adminAPIHandlers) SetConfigHandler(w http.ResponseWriter, r *http
 		return
 	}
 
-	var config serverConfigV20
+	var config serverConfigCurrent
```

Comment:
With this approach, how will be able to import config file with version 20 when we move to version 21? IIUC, when we release minio with config version 21, we won't have a type which is capable of importing config file version 20 during migration.

Reply:
Added additional step:
```
Copy serverConfigCurrent struct to serverConfigVN (in config-old.go)
```

Check the new steps:
```
// Steps to move from version N to version N+1
// 1. Copy serverConfigCurrent struct to serverConfigVN (in config-old.go)
// 2. Change the necessary serverConfigCurrent fields to accomodate new changes.
// 3. Change the configCurrentVersion to "N+1"
// 4. Add new migration function (ex. migrateVNToVN+1())
// 5. Add migrateVNToVN+1() to migrateConfig() in config-migrate.go
// 6. Make changes in config-current_test.go for any test change
```

Reply:
To verify this steps in practice, we could move serverConfig19 to config-old.go and follow the remaining steps explained in the above comment. This would leave a working example besides the above comments for future developers.

Reply:
@krishnasrinivas can you incorporate @krisis recommendation? - We can move ahead and merge this PR with that change in place.